support/doc/tools.md

```diff
@@ -42,20 +41,12 @@ Clone the PeerTube repo to get the latest version (even if you are on your PeerT
 
 ```
 $ git clone https://github.com/Chocobozzz/PeerTube.git
-$ CLONE="$(pwd)/PeerTube"
+$ CLONE="$(pwd)/PeerTube && cd $CLONE"
 ```
 
-Run ``yarn install --pure-lockfile``
-```
-$ cd ${CLONE}
-$ yarn install --pure-lockfile
-```
+Run `yarn install --pure-lockfile` or `NOCLIENT=1 yarn install --pure-lockfile` if you wish to save space by not having dependencies for the web interface of PeerTube. Likewise if you are on a production installation that has already its server compiled, you may add the `--production` flag and skip the next command.
```

Comment:
Remove `yarn install --pure-lockfile` since it's better to use the `NOCLIENT` version

Reply:
Okay.